### PR TITLE
setup.cfg: add `metadata.project_urls`

### PR DESCRIPTION
**Pull request by generalmimon**
Fix https://github.com/kaitai-io/kaitai_struct_python_runtime/issues/69

According to a project example https://pypi.org/project/PyScaffold/4.3/ + https://github.com/pyscaffold/pyscaffold/blob/v4.3/setup.cfg#L7-L15, the "Homepage" link pulled from `metadata.url` apparently stays even if I add `project_urls`, so all links at PyPI resulting from this change should be (I'm not sure about their order, but again links on https://pypi.org/project/PyScaffold/4.3/ seem to be sorted alphabetically except the implicit "Homepage" link at the beginning):

- [Homepage](https://kaitai.io)
- [Documentation](https://doc.kaitai.io/)
- [Gitter](https://gitter.im/kaitai_struct/Lobby)
- [Source](https://github.com/kaitai-io/kaitai_struct_python_runtime)
- [Tracker](https://github.com/kaitai-io/kaitai_struct_python_runtime/issues)

Currently (see https://pypi.org/project/kaitaistruct/0.10/) the project links are just:

- [Homepage](https://kaitai.io)

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -4,6 +4,11 @@ version = attr: kaitaistruct.__version__
 author = Kaitai Project
 author_email = greycat@kaitai.io
 url = https://kaitai.io
+project_urls =
+    Source = https://github.com/kaitai-io/kaitai_struct_python_runtime
+    Tracker = https://github.com/kaitai-io/kaitai_struct_python_runtime/issues
```
Comment (generalmimon) [inline setup.cfg:9]: We can also add some "Documentation" link, but I'm not sure if it should link to https://doc.kaitai.io/, https://doc.kaitai.io/lang_python.html or somewhere else.

**Comment by KOLANICH** [comment]
IMHO it'd be a bit redundant to add the links `source code` and `issue tracker` since everyone knows that source code hostings (GitHub currently) are used for that. So IMHO 1 homepage link to this repo is enough. The links to the docs ... you don't have to limit oneself. Proposal: 1 link to the docs for python runtime, and another one to the guide to KS language.

**Comment by KOLANICH** [comment]
Also: `setup.cfg` is deprecated, convert it to `pyproject.toml` using `ini2toml` tool.

**Review by dgelessus — APPROVED** [review]
The list of links looks good to me. I personally wouldn't have added a Twitter link here (I think it's not all that relevant when coming from PyPI), but I have no problem with keeping it if the list doesn't become too long.

A link to the docs sounds good, but I have no strong opinion about linking to the main docs or the Python notes (the reader can always click through to the other docs if needed).

**Comment by dgelessus** [comment]
> Also: `setup.cfg` is deprecated,

From where do you have that information? The [setuptools docs for `setup.cfg`](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html) don't mention anything about it being deprecated. All I can find is [pypa/setuptools/milestone/7](https://github.com/pypa/setuptools/milestone/7) about *eventually* deprecating `setup.cfg`, but that hasn't happened yet. The setuptools maintainers also mentioned in [multiple](https://github.com/pypa/setuptools/issues/1688#issuecomment-471271447) [places](https://github.com/pypa/setuptools/issues/1160#issuecomment-549054957) that the deprecation period would probably be "forever", so there's no rush.

> convert it to `pyproject.toml` using `ini2toml` tool.

That's still not an option. [As before](https://github.com/kaitai-io/kaitai_struct_python_runtime/pull/39#pullrequestreview-937134754), `pyproject.toml`-based configuration requires setuptools 61.0.0, which requires Python 3.7, and we still supports version before that.

**Comment by generalmimon** [comment]
@dgelessus:

> The list of links looks good to me. I personally wouldn't have added a Twitter link here (I think it's not all that relevant when coming from PyPI)

I agree that it's not that relevant for the Python runtime, and it's not clear what a Python user would find there.

**Comment by generalmimon** [comment]
@dgelessus:

> A link to the docs sounds good, but I have no strong opinion about linking to the main docs or the Python notes (the reader can always click through to the other docs if needed).

That's true, so I think the main docs are fine.

**Pull request merged**